Prospress/ruleset.xml

```diff
@@ -91,6 +91,7 @@
 		<properties>
 			<!-- e.g. body_class, the_content, the_excerpt -->
 			<property name="customAutoEscapedFunctions" type="array" value="woocommerce_wp_select"/>
+			<property name="customAutoEscapedFunctions" type="array" value="wcs_help_tip"/>
```

Comment:
Yep looks like we can. Source: https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/issues/379#issuecomment-202038133 (bottom of that comment).